### ISSUE TITLE
[OAKTrust] Full publication item example page

### ISSUE THREAD

**Issue by devangm** [TAMULib/dspace-angular#206]
**URL:** https://oaktrust-dev.library.tamu.edu/items/35bb9e2e-afcb-4115-9b01-c83b4d8ca1a8/full
**WCAG 2.1 AA compliant? :** No
**Violation:** 4.1.2 Name, Role, Value (A)
**Issues:** broken ARIA menu
**Notes:** There seems to be a non-interactable ARIA menu that doesn't need to be there.
